src/control/cmd/daos_agent/mgmt_rpc.go

```diff
@@ -170,10 +176,20 @@ func (mod *mgmtModule) handleGetAttachInfo(reqb []byte, pid int32) ([]byte, erro
 		return nil, err
 	}
 
-	return mod.aiCache.getResponse(numaNode)
+	cacheResp, err := mod.aiCache.getResponse(numaNode)
+	if err != nil {
+		return nil, err
+	}
+
+	mod.monitor.submitRequest(pid, drpc.MethodGetAttachInfo)
+
+	return cacheResp, err
 }
 
-func (mod *mgmtModule) handleDisconnect(reqb []byte, pid int32) ([]byte, error) {
-	mod.log.Debugf("Disconnect is currently not implemented")
-	return nil, nil
+// handleDisconnect crafts a new request for the process monitor to inform the
+// monitor that a process is exiting. Even though the process is terminating
+// cleanly disconnect will inform the control plane of any outstanding handles
+// that the process held open.
+func (mod *mgmtModule) handleDisconnect(reqb []byte, pid int32) {
```

Comment:
Looks like reqb isn't used here. If it's not needed, could it be removed entirely?

Reply:
Removed that.